pages/products/kpi-options.tsx

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import type { NextPage } from "next";
+import ProductTemplate, { Section } from "../../components/ProductPageTemplate";
+
+const sections = [
+  {
+    type: Section.HeroCarousel as Section.HeroCarousel,
+    title: "KPI Options",
+    image: {
+      src: "/images/products/powered_by_uma.png",
+      alt: "UMA_logo",
+    },
+    carousel: {
+      prefix: "Smart airdrops motivate your community to",
+      variations: [
+        "migrate liquidity",
+        "recruit friends",
+        "establish partnerships",
+        "increase adoption",
+        "create content",
+        "proseltyze relentlessly",
+      ],
+    },
+  },
+  {
+    type: Section.KeyFeatures as Section.KeyFeatures,
+    features: [
+      {
+        title: "Made-to-order Targets",
+        details: "Define the goal that matters most to your DAO",
+        image: {
+          src: "/images/products/kpi-target.svg",
+          alt: "Made-to-order Targets",
+        },
+      },
+      {
+        title: "Align Incentives",
+        details: "Get your community on the same page",
+        image: {
+          src: "/images/products/kpi-incentives.svg",
+          alt: "Align Incentives",
+        },
+      },
+      {
+        title: "Treasury Efficiency",
+        details: "Dump-proof airdrop",
+        image: {
+          src: "/images/products/kpi-parachute.svg",
+          alt: "Treasury Efficiency",
+        },
+      },
+      {
+        title: "Standard Tokens",
+        details: "ERC20 tokens with all the composability that comes with it",
+        image: {
+          src: "/images/products/kpi-tokens.svg",
+          alt: "Standard Tokens",
+        },
+      },
+    ],
+  },
+  {
+    type: Section.Overview as Section.Overview,
+    title: "What is it?",
+    image: {
+      src: "/images/products/kpi-mobile-img2.png",
+      alt: "Example KPI UI",
+      label:
+        "Example UI for claiming and viewing KPI Options tokens for your community",
```

Comment:
<img width="359" alt="Screen Shot 2022-03-18 at 1 59 46 PM" src="https://user-images.githubusercontent.com/71620056/159058122-7c644358-4234-4ec8-9b4b-c1862213544d.png">


Lets update the color of the font "Example UI...." to that of the above designs